### PR TITLE
[VGNWB-311] Added pagination for collections

### PR DESCRIPTION
**Pull request by Rudi114**

### CODE REVIEW

**Review by tjquinn1 — CHANGES_REQUESTED** [review]
It looks good, lets just make the rows per page text a little bigger.

**Comment by Rudi114** [comment]
Font Size increased

File: src/components/texts/RepoCollections.vue [hunk 1]
```diff
@@ -24,5 +36,18 @@ export default class RepoCollections extends Vue {
...
 }
-</script>
\ No newline at end of file
+</script>
+
+<style scoped>
+.row-select {
+	padding-right: 31px;
+}
+.rows {
```
Comment (tjquinn1) [inline src/components/texts/RepoCollections.vue]: This needs to be named something else. You should be careful with class names. When you use grids in vuetify it just creates a div with the class row. Had you used rows in this you would have changed all the rows. This is usually the case with most css framworks as well e.g. bootstrap. I have added a screenshot so you can see what the rendered rows look like, it maybe helpful to look through the code that is rendered in the browers and read some documentation.

Screen Shot 2020-03-02 at 1 06 41 PM

Comment (tjquinn1) [reply src/components/texts/RepoCollections.vue]: Also lets take the front size down just a little as well

**Review by tjquinn1 — CHANGES_REQUESTED** [review]

**Review by tjquinn1 — APPROVED** [review]

**Pull request merged**